test/integration/full/test-webdriver.js

```diff
@@ -153,14 +153,12 @@ function start(options) {
   options.browser =
     options.browser === 'edge' ? 'MicrosoftEdge' : options.browser;
 
-  const testUrls = globby
-    .sync([
-      'test/integration/full/**/*.{html,xhtml}',
-      '!**/frames/**/*.{html,xhtml}'
-    ])
-    .map(url => {
-      return 'http://localhost:9876/' + url;
-    });
+  const testUrls = globSync([
+    'test/integration/full/**/*.{html,xhtml}',
+    '!**/frames/**/*.{html,xhtml}'
```

Comment:
The test failure is happening because `glob` stopped supporting `!` prefixes for negation in v6; instead, use its `ignore` option. See https://github.com/isaacs/node-glob#comments-and-negation

Reply:
Oh right. That's what I get for not looking at the exact glob pattern